_terms/legal-eula.md

```diff
@@ -44,38 +44,104 @@ last_update: 01/10/2016
+5\. Termination of the License
+---
+
+5.1	The license to use the Software, Service or App hereunder shall expire or terminate in accordance with the Terms. Without prejudice to the foregoing, the license shall be terminable by SixSq on written notice if: (a) the continued use or possession of the Software, Service or App by Customer or End-User infringes SixSq’s or a third party’s rights; or (b) Customer or End-User commits a material or persistent breach of this EULA or the Terms which Customer and/or End-User fails to remedy (if remediable) within 10 days of written notice from SixSq.
+
+5.2	Without limiting its other rights or remedies, SixSq may terminate this license and/or EULA with immediate effect on notice to the End-User if SixSq has not received full payment for the Products on the due date for payment, or if a false or misleading information is provided in respect of the end–customer.
+
+5.3	Upon termination of this license and EULA for any reason: (a) all rights granted to the End-User under this EULA shall cease; (b) the End-User must cease all activities authorised by this EULA; (c) the End-User must immediately pay to SixSq any sums due to SixSq under this EULA; (d) the End-User must immediately stop using the Service and Apps and delete or remove the Software from all computer equipment in the End-User’s possession, and at SixSq’s option, immediately destroy or return to SixSq all copies of the Software and any SixSq Materials (and any deliverables which have not been fully paid for then in the End-User’s possession), and, in the case of destruction, certify to SixSq that the End-User has done so; (e) the accrued rights, remedies, obligations and liabilities of the parties as at expiry or termination shall not be affected, including the right to claim damages in respect of any breach of this EULA which existed at or before the date of termination or expiry; and (f) Clauses which expressly or by implication have effect after termination shall continue in full force and effect.
+
+6\. Service Level Agreement (SLA)
+---
+
+6.1 End-Users purchase or subscribe to a level of service when using the Service. Several SLAs are be defined in the EULA, unless specified directly in the Order Confirmation.
+
+6.2 By default, the *Bronze Service Level Agreement* applies if none other are specified in the Order Confirmation. The Bronze SLA includes: (a) 24 hours a day and 7 days a week automated monitoring service; (b) human support service available during normal working days; (c) the human support service is not available during the official public holidays in Geneva; (d) End-users must report serious issues by creating a ticket, sending an email to [support@sixsq.com](mailto:support@sixsq.com) or using an in Service feature (if available); 
+
+6.3 SixSq shall use commercially reasonable efforts to make the Services available 24 hours a day, 7 days a week and according to the corresponding Services pledged availability (see 6.4), except for:
```

Comment:
probably you could mention third party providers as well, in the sense that if Exoscale goes down for 1 day, we cannot be liable for that

Reply:
Done